utils/build/docker/python/flask/app.py

```diff
@@ -166,6 +173,90 @@ def dbm():
     return Response(f"Integration is not supported: {integration}", 406)
 
 
+@app.route("/dsm")
+def dsm():
+    logging.basicConfig(
+        format = '%(asctime)s %(levelname)-8s %(message)s',
+        level = logging.INFO,
+        datefmt = '%Y-%m-%d %H:%M:%S')
+    topic = "dsm-system-tests-queue"
+    consumer_group = "testgroup1"
+    logging.info(os.environ)
+
+    def delivery_report(err, msg):
+        logging.info("[kafka] start delivery_report")
+        if err is not None:
+            logging.info(f"[kafka] Message delivery failed: {err}")
+        elif msg is not None:
+            logging.info("[kafka] Delivered to partition:")
+            logging.info(msg.partition())
+            logging.info(f"[kafka] Message delivered to {msg.partition()}")
```

Comment:
This line causes the following error:
[2023-09-26 22:53:10 +0000] [10] [ERROR] Worker (pid:29) was sent SIGSEGV!